src/i18n/translations/en.ts

```diff
@@ -650,4 +650,10 @@ export const messages: MessagesKey = {
   mapLoadingStopsDataText: 'Loading stops...',
   locateStopPoint: 'Locate',
   locateStopPointTooltip: 'Locate on map',
+  exportsDownloadUnsupportedBrowserTitle: 'Browser Not Supported',
```

Comment:
Swedish: Webbläsaren stöds inte

Reply:
Finnish: _Tätä selainta ei tueta_

Literally "this browser is not supported". You can also drop "_Tätä_", but it feels more natural to include it as it brings context to the translation.